src/main/drivers/pwm_output.c

```diff
@@ -397,7 +395,9 @@ void pwmWriteDshotCommand(uint8_t index, uint8_t command)
         for (; repeats; repeats--) {
             motor->requestTelemetry = true;
             pwmWriteDshotInt(index, command);
-            pwmCompleteMotorUpdate(0);
+            if (pwmMotorsEnabled) {
```

Comment:
Yes, it looked wrong to me. This is how the existing code is, see https://github.com/betaflight/betaflight/blob/master/src/main/drivers/pwm_output_dshot.c#L73-L85 it's just the guard was previously in `pwmCompleteDshotMotorUpdate`.